docs/examples/modal/mobileExample.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Flex, Layer, Modal, Box, Text, DeviceTypeProvider, Button } from 'gestalt';
+
+export default function Example(): Node {
+  const [showModal, setShowModal] = React.useState(false);
+
+  return showModal ? (
```

Comment:
Probably no need to hide the button when the modal is showing, right? It should overlay the button?

Reply:
Indeed, copied from sidenavigation, but yeah, good point

DONE